components/automate-ui/src/app/pages/project/list/project-list.component.html

```diff
@@ -14,37 +14,57 @@
     </section>
 
     <section *ngIf="(iamMinorVersion$ | async) === 'v1'" class="page-body">
-      <chef-toolbar>
+      <ng-container *ngIf="(sortedProjects$ | async)?.length > 0">
+        <chef-toolbar>
+          <app-authorized [allOf]="['/iam/v2beta/projects', 'post']">
+            <chef-button id="create-button" primary (click)="openCreateModal()">Create Project</chef-button>
+          </app-authorized>
+        </chef-toolbar>
+        <app-authorized [allOf]="['/iam/v2beta/projects', 'get']">
+          <chef-table>
+            <chef-thead>
+              <chef-tr>
+                <chef-th>Name</chef-th>
+                <chef-th>Type</chef-th>
+                <chef-th class="controls"></chef-th>
+              </chef-tr>
+            </chef-thead>
+            <chef-tbody>
+              <chef-tr *ngFor="let project of sortedProjects$ | async">
+                <chef-td>
+                  <a [routerLink]="['/settings/projects', project.id]">{{ project.name }}</a>
+                </chef-td>
+                <chef-td>{{ project.type | iamType }}</chef-td>
+                <chef-td class="controls">
+                  <app-authorized [allOf]="['/iam/v2beta/projects/{id}', 'delete', project.id]">
+                    <chef-control-menu>
+                      <chef-option (click)="startProjectDelete(project)">Delete Project</chef-option>
+                    </chef-control-menu>
+                  </app-authorized>
+                </chef-td>
+              </chef-tr>
+            </chef-tbody>
+          </chef-table>
+        </app-authorized>
+      </ng-container>
+      <ng-container *ngIf="(sortedProjects$ | async)?.length === 0">
         <app-authorized [allOf]="['/iam/v2beta/projects', 'post']">
-          <chef-button id="create-button" primary (click)="openCreateModal()">Create Project</chef-button>
+          <div class="empty-case-container">
+            <h1>Create the first project to get started!</h1>
```

Comment:
sorry! didnt look closely enough, these should be paragraphs, not headings. its important for semantics and accessibility.